### DISCUSSION TITLE
RaspPi 3b+ Mycroft Image can't install

### DISCUSSION THREAD

**Discussion by pierreode** [dtcooper/raspotify#603] [Q&A] [answered]
Hello

I've been trying to set up Raspotify for a few hours now but keep running into the same hurdle.

This morning I started setting up Mycroft with the latest stable image on their website. I have it working the way I want it.

Next step for me was to get it working with Spotify. Following [this](https://market.mycroft.ai/skills/06e089a9-bcd4-498b-99ce-779656667847) guide, I discovered Respotify was the way to go. Then using the [Basic Seup](https://github.com/dtcooper/raspotify/wiki/Basic-Setup-Guide) guide, I enter the command `sudo apt-get -y install curl && curl -sL https://dtcooper.github.io/raspotify/install.sh | sh` and it happily says that curl is already up to date, but that all the other dependancies are out of date.
```
Unmet minimum required package version(s):

libc6 >= 2.31 is required but 2.28-10+rpt2+rpi1+deb10u1 is installed.
coreutils >= 8.32 is required but 8.30-3 is installed.
systemd >= 247.3 is required but 241-7~deb10u8+rpi1 is installed.
libasound2 >= 1.2.4 is required but 1.1.8-1+rpt2 is installed.
alsa-utils >= 1.2.4 is required but 1.1.8-2+rpt1 is installed.
libpulse0 >= 14.2 is required but 12.2-4+deb10u1+rpt3 is installed.
```

I do a `ldd --version` and can see that it is the version that is required.
```
ldd (Debian GLIBC 2.28-10+rpt2+rpi1+deb10u1) 2.28
```

I have done a `sudo apt-get update` and `sudo apt-get upgrade` and am currently running the latest version of Debian(buster).

At this point I am not too sure where to go from here so some help would be great!

**Comment by JasonLG1979** [comment, marked as answer]
[Debian 11 (Bullseye)](https://www.debian.org/releases/bullseye/) is the latest stable version not [10 (Buster)](https://www.debian.org/releases/buster/).

That guide is also EXTREMELY out of date. 

Duplicate of https://github.com/dtcooper/raspotify/issues/529

**Reply by pierreode** [reply]
Oh, thanks for that! I'll give that a shot!
